build.properties

```diff
@@ -6,4 +6,4 @@ project.revision=7.1-SNAPSHOT
 
 junit.maxmemory=500M
 
-build.modules.list=hive-jdbc,hdp23,hdp24,cdh57,cdh58,mapr410,mapr510,emr310,emr41,emr46
+build.modules.list=hive-jdbc,hdp23,hdp24,hdp25,cdh57,cdh58,mapr410,mapr510,emr310,emr41,emr46
```

Comment:
Also, has the build properties that need to be created been reviewed by @buildguy